.gitignore

```diff
@@ -5,3 +5,4 @@ vendor
 *.log
 tags
 .DS_Store
+.tern-project
```

Comment:
I hadn't even thought of that.  It's in composer's `.gitignore` so I just added it here for ease.